doc/DataPlaneCodeGeneration/AzureSDKCodeGeneration_DataPlane_Quickstart.md

```diff
@@ -2,243 +2,33 @@
-    - [Add Convenience APIs](#add-convenience-apis)
-    - [APIView](#apiview)
-
-<!-- /TOC -->
-
 ## Prerequisites
 
-- Install Visual Studio 2022 (Community or higher) and make sure you have the [latest updates](https://www.visualstudio.com/).
-  - Need at least .NET Framework 4.6.1 and 4.7 development tools
-  - Install the **.NET Core cross-platform development** workloads in Visual Studio
-- Install **.NET 6.0 SDK** for your specific platform. (or a higher version)
-- Install the latest version of [git](https://git-scm.com/downloads)
-- Install [PowerShell](https://docs.microsoft.com/powershell/scripting/install/installing-powershell), version 7 or higher.
-- Install [NodeJS](https://nodejs.org/) (16.x.x or above).
-
-## Setup your repository
-
-- Fork and clone the [azure-sdk-for-net](https://github.com/Azure/azure-sdk-for-net) repository. Instructions for doing so can be found in the [.NET CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md).
-- Create a branch to work in.
-
-## Create starter package  
-
-For this guide, we'll create a getting started project in a branch of your fork of `azure-sdk-for-net` repo. The started project will be under `sdk\<servie name>\<package name>` directory of `azure-sdk-for-net` repo. The package will contain several folders and files (see following). Please refer to [sdk-directory-layout](https://github.com/Azure/azure-sdk/blob/main/docs/policies/repostructure.md#sdk-directory-layout) for detail information.
-
-```text
-sdk\<service name>\<package name>\README.md
-sdk\<service name>\<package name>\api
-sdk\<service name>\<package name>\perf
-sdk\<service name>\<package name>\samples
-sdk\<service name>\<package name>\src
-sdk\<service name>\<package name>\stress
-sdk\<service name>\<package name>\tests
-sdk\<service name>\<package name>\CHANGELOG.md
-sdk\<service name>\<package name>\<package name>.sln
-```
-
-- `<service name>` - Should be the short name for the azure service. e.g. deviceupdate
-- `<package name>` -  Should be the name of the shipping package, or an abbreviation that distinguishes the given shipping artifact for the given service. It will be `Azure.<group>.<service>`, e.g. Azure.IoT.DeviceUpdate
-
-### Use Cadl as Input
-
-#### Create Cadl project
-  
-  You can download existing cadl project from [azure-rest-api-specs](https://github.com/Azure/azure-rest-api-specs) repo or you can follow following steps to create a new cadl project.
-
-  ***Initialize Cadl Project***
-
-  Follow [Cadl Getting Start](https://github.com/microsoft/cadl/#using-node--npm) to initialize your Cadl project.
-
-  Make sure `npx cadl compile .` runs correctly.
-
-  ***Add cadl-csharp emitter***
-
-  Modify package.json, add one line under dependencies:
-
-```diff
-        "dependencies": {
-          "@cadl-lang/compiler": "^0.37.0",
-          "@cadl-lang/rest": "^0.19.0",
-          "@azure-tools/cadl-azure-core": "^0.9.0",
-+         "@azure-tools/cadl-csharp": "0.1.8"
```

Comment:
shall we update this dependency by using typespec-csharp?

Reply:
This line is to be removed.